src/command_modules/azure-cli-deploymentmanager/azure/cli/command_modules/deploymentmanager/tests/latest/test_deploymentmanager_scenario.py

```diff
@@ -0,0 +1,620 @@
+        self.replace_string(storageAcountReplacementSymbol, stgAcctForTemplate, parametersCopyArtifactSourceRelativePath)
+
+        storage_account_info = self.get_stg_account_info(resource_group_name, storage_account_name)
+        self.create_container(storage_account_info, container_name)
+
+        self.upload_blob(storage_account_info, container_name, parametersArtifactSourceRelativePath, parameters_file_name)
+        self.upload_blob(storage_account_info, container_name, parametersCopyArtifactSourceRelativePath, params_copy_file_name)
+        self.upload_blob(storage_account_info, container_name, templateArtifactSourceRelativePath, template_file_name)
+        self.upload_blob(storage_account_info, container_name, templateCopyArtifactSourceRelativePath, template_copy_file_name)
+        self.upload_blob(storage_account_info, container_name, invalidParametersArtifactSourceRelativePath, invalid_parameters_file_name)
+
+    def delete_artifact_source(self, resource_group_name, artifact_source_name):
+        self.kwargs = {
+            'rg': resource_group_name,
+            'name': artifact_source_name,
+        }
+        self.cmd('deploymentmanager artifact-source delete -n {name} -g {rg} --yes')
+
+        with self.assertRaisesRegexp(CloudError, 'not found'):
+            self.cmd('deploymentmanager artifact-source show -n {name} -g {rg}')
+
+    def upload_blob(self, storage_account_info, storage_container_name, file_path, file_name):
+        abs_file_path = os.path.join(TEST_DIR, file_path).replace('\\', '\\\\')
+        blob_path = artifact_root + "/" + file_name
+        self.storage_cmd('storage blob upload -c {} -n {} -f "{}" -t block', storage_account_info,
+                         storage_container_name, blob_path, abs_file_path)
+
+    def get_stg_account_key(self, group, name):
+
+        template = 'storage account keys list -n {} -g {} --query "[0].value" -otsv'
+        return self.cmd(template.format(name, group)).output
+
+    def get_stg_account_info(self, group, name):
+        """Returns the storage account name and key in a tuple"""
+        return name, self.get_stg_account_key(group, name)
+
+    def create_container(self, account_info, container_name):
+        self.storage_cmd('storage container create -n {}', account_info, container_name)
+        return container_name
+
+    def storage_cmd(self, cmd, account_info, *args):
+        cmd = cmd.format(*args)
+        cmd = '{} --account-name {} --account-key {}'.format(cmd, *account_info)
+        return self.cmd(cmd)
+
+    def replace_string(self, replacement_symbol, replacement_value, filePath):
+        is_playback = os.path.exists(self.recording_file)
+        if not is_playback:
+            with fileinput.FileInput(filePath, inplace=True) as file:
+                for line in file:
+                    print(line.replace(replacement_symbol, replacement_value), end='')
```

Comment:
CI is failing because this line is incompatible with Python2.

It's an easy fix though! Just add 
``` Python
from __future__ import print_function
```
to the top of this file.

You can see other tips for supporting both Python 2 & 3 here: https://python-future.org/compatible_idioms.html

Reply:
Done, thanks @marstr